crates/sparrow-runtime/src/read/parquet_file.rs

```diff
@@ -36,7 +35,7 @@ pub enum Error {
     ReadingParquetFile,
 }
 
-const BATCH_SIZE_ROWS: usize = 100_000;
+const BATCH_SIZE_ROWS: usize = 4_096;
```

Comment:
Why this change?

Reply:
Read on arrow mailing list that batches of 4k or 8k rows were preferred.